### ISSUE TITLE
[Bug]: btcpay server doesnt show qr 

### ISSUE THREAD

**Issue by mannder** [btcpayserver/btcpayserver#6028]
### What is your BTCPay version?

v1.13.1+ca55e1f30 

### How did you deploy BTCPay Server?

installed in Umbrel 0.5.4

### What happened?

i installed btcpay server on a shopify store, and when i try to checkout a product, the payment popup doesn,t show, and in the store shows the new order made.

### How did you encounter this bug?

when i try to buy a product on a shopify store and paying with btcpay server

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by ndeet** [comment]
Hi, I had the same issue recently when Kraken as rate provider was down, was this also the case for you or do you face another issue? So try to change the rate provider of your store to e.g. CoinGecko, if the problem persists or is resolved please let us know.

Also to be sure we are on the same page, you clicked on the Pay with Bitcoin button on the order confirmation page?

You could try to open javascript console and see if there is some error showing up.

**Comment by bcavilla** [comment]
Hi, thanks for the help, i had Kraken as rate provider, i changed it to Coingecko but it did the same, doesnt show the invoice, should i reconfigure it again?
i clicked in pay with Bitcoin BTCpay server at the checkout page (see atach) 
<img width="642" alt="Captura de pantalla 2024-06-04 a la(s) 08 43 03" src="https://github.com/btcpayserver/btcpayserver/assets/110301928/95a513c3-c153-4da7-8e6a-57a72fc68c0d">
, the button never apears.

**Comment by petzsch** [comment]
Could you please provide the following info:

- what browser was used (some like brave can be very restrictive)
- if possible please share a link to your store so it can be tested also also possible JS errors be inspected
- Is there an invoice created in BTCPay at all? If so; Anything "red" in the invoice details table when clicking on the Invoice ID in btcpay?

**Comment by bcavilla** [comment]
Hi, i tried Brave, Firefox and Firewolf, even safari on iPhone
the store is tienda21.com (not online yet, i removed the password so you can see it)
there is not an invoice

**Comment by petzsch** [comment]
perhaps @ndeet can have a look at the JS stuff. I'm not to familiar with it.
Tried a test order and landed right on the confirmation page without payment.

**Comment by ndeet** [comment]
@bcavilla something or more is off on your store, the script you should embed in the checkout scripts section points to a local network ip address while it should be some domain instead like btcpay.yourstore.com but it is this at the moment:
> http://192.168.0.20:3003/stores/Hi6LeErshsQv9PmRrKJvBSjy3HpDqvRJrYB3ZkKyYKdi/plugins/shopify/shopify.js

Also there is a weird error seemingly unrelated on the thank you page I basically see order number and thank you, only after some while the rest like the map shows up. So maybe there is something wrong with your theme or other modifications you may did? 
> Refused to apply style from 'https://tienda21.com/private_access_tokens?id=6194811aaa81ba089a11b2b7728a11ed' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

Anyway, to fix the BTCPay button you need make sure your BTCPay Server is reachable from the internet and has a subdomain or domain assigned otherwise it won't work.

**Comment by bcavilla** [comment]
hi,
It points to a ip adress because is how i enter to my btcpay server, its in my umbrel btc node. how could i change it?
how can i do my btcpay reachable from the internet?

**Comment by dennisreimann** [comment]
> how can i do my btcpay reachable from the internet?

Imho the best way to set it up is the [reverse ssh proxy](https://docs.btcpayserver.org/Deployment/ReverseSSHtunnel/) approach. You could also try the [Cloudflare tunnel](https://docs.btcpayserver.org/Docker/cloudflare-tunnel/), but there are issues with the Lightning connections, because CF doesn't tunnel the LN port `9375`. More on that in #6020.

**Comment by ndeet** [comment]
Also this guide for Ubrel via Tailscale VPN https://orange.surf/public-btcpay-umbrel-tailscale/
